.github/workflows/build.yml

```
@@ -0,0 +1,305 @@
+name: build
+
+env:
+  SGX_MODE: HW
+  IAS_MODE: PROD
+  RUST_BACKTRACE: full
```

Optional: Use 1. I don't find the output of `RUST_BACKTRACE=full` more informative than `RUST_BACKTRACE=1`

```suggestion
  RUST_BACKTRACE: 1
```